djangocms_versioning/admin.py

```diff
@@ -341,10 +344,15 @@ def edit_redirect_view(self, request, object_id):
         elif version.state != DRAFT:
             raise Http404
         # Redirect
-        url = reverse('admin:{app}_{model}_change'.format(
-            app=version.content._meta.app_label,
-            model=version.content._meta.model_name,
-        ), args=(version.content.pk,))
+        # If the object is editable the cms editable view should be used, with the toolbar.
+        if is_editable_model(version.content):
```

Comment:
`is_editable_model` takes a class, and although it seems to work for model instances when checking if model has a relation to Placeholder, it doesn't work with checking its admin for FrontendEditableAdminMixin inheritance.